packages/monitors/index.js

```diff
@@ -274,6 +274,7 @@ async function run({
           message: "End of serverless execution loop - terminating process"
         });
         await waitForLogger(logger);
+        await delay(2); // waitForLogger does not always work 100% correctly in serverless. add a delay to ensure logs are captured upstream.
```

Comment:
Should we parameterize in `monitorConfig`

Reply:
> Nice catches, left some comments about where to parameterize hard coded values

right now I don't think we should parameterize them. Not really sure of a situation where we'd want it to not be the full 2-second delay. What scenario are you picturing where we'd want to change it?